ironfish/src/mining/stratum/messages.ts

```diff
@@ -43,6 +52,14 @@ export const StratumMessageSchema: yup.ObjectSchema<StratumMessage> = yup
   })
   .required()
 
+export const MiningDisconnectMessageSchema: yup.ObjectSchema<MiningDisconnectMessage> = yup
+  .object({
+    reason: yup.string().optional(),
+    versionExpected: yup.number().optional(),
+    bannedUntil: yup.number().optional(),
+  })
```

Comment:
Not sure if intentional, but on L17, we add `message` to the `MiningDisconnectMessage`, but we don't add it to the schema here. I think we want to add it here, then we can display it on the client's side right?